rpc_jobs/irr_role_test.yml

```diff
@@ -0,0 +1,94 @@
+- project:
+    name: 'RPC-IRR-Jobs'
+    # Note: branch is the branch for periodics to build
+    #       branches is the branch pattern to match for PR Jobs.
+    repo:
+      # IRR REPOS ARE ADDED HERE
+      - github:
+          url: https://github.com/rcbops/rpc-maas
```

Comment:
I *think* that it will need to be something like this instead:

```
  repo:
    # IRR REPOS ARE ADDED HERE
    - rpc_maas:
      properties:
        - github:
            url: https://github.com/rcbops/rpc-maas
```